osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -239,6 +239,11 @@ public override void RequestResume(Action continueResume)
                 continueResume();
         }
 
+        public override void CancelResume()
+        {
+            ResumeOverlay.Hide();
```

Comment:
I think this needs a null check, `ResumeOverlay` doesn't look to be guaranteed to be non-null (see [line 199 where it is constructed](https://github.com/ppy/osu/blob/b0e21c2749d24d58de0b4c92569244bdeed77460/osu.Game/Rulesets/UI/DrawableRuleset.cs#L199), and then also [line 232 when a resume is requested](https://github.com/ppy/osu/blob/b0e21c2749d24d58de0b4c92569244bdeed77460/osu.Game/Rulesets/UI/DrawableRuleset.cs#L232)).

Reply:
Thanks! I didn't realize that, I added a null check just now!